Source/NuGet.config

```diff
@@ -1,8 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="Identity* Dev Feed" value="https://www.myget.org/F/identity/" />
+    <add key="Platform Packages" value="https://repository.domain.com.au/nuget/platform/" />
+    <add key="Domain" value="https://repository.domain.com.au/nuget/domain/" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="DomainUsers" value="https://repository.domain.com.au/nuget/users/" />
```

Comment:
Is this needed?

Reply:
Yes. Otherwise, the build script was not able to find IdentityServer3 config from https://repository.domain.com.au/nuget/platform/. Ya. perhaps we may don't need the other feeds. Let me try removing the others..

Reply:
Actually, none of these changes are needed. Earlier, it was not being able to find the specified version and I thought changing the .configs fixed it. Thanks for pointing out!